templates/modern-solutions/dop/Makefile

```diff
@@ -0,0 +1,16 @@
+all: dop payload
+
+.PHONY: clean
+
+CC = clang
+CFLAGS = -static
+#CFLAGS = -fsanitize=safe-stack
```

Comment:
Why is this here? Is it part of the challenge to also use the safe-stack sanitizer?

Reply:
Yes. The safe-stack sanitizer is the last step of the DOP challenge.